src/config/index.ts

```diff
@@ -62,5 +62,5 @@ export const config = {
     mockSep24: false,
     overwriteMinimumTransferAmount: false,
   },
-  telegramUrl: 'https://t.me/vortex_fi',
+  telegramUrl: 'https://forms.gle/bgH4XTTbQ3YbwQ3t7',
```

Comment:
Nit: let's rename this to `supportUrl` then, if this is not related to telegram anymore.
```suggestion
  supportUrl: 'https://forms.gle/bgH4XTTbQ3YbwQ3t7',
```

Reply:
Good point, it is not even telegram now anymore but a Google form.